### ISSUE TITLE
Workaround is used to remove unnecessary semicolons

### ISSUE THREAD

**Issue by bulyshko** [bulyshko/esformatter-semicolons#2]
Currently, a [workaround](https://github.com/bulyshko/esformatter-semicolons/blob/master/index.js#L37-L50) is used. I was trying to simply remove unnecessary semicolons in many different ways, but I was getting the same bug:
### Sample input:

``` javascript
var foo = 'bar',
    bar = 'qux'
    ;;;

function abc() {
    return 123;
}
```
### Output without a workaround:

``` javascript
var foo = 'bar',
    bar = 'qux';

    function abc() {
        return 123;
        }
```

Look for an alternative solution.

**Comment by millermedeiros** [comment]
yes, there is a bug on esformatter itself. I just added a new feature called `pipe` to solve this kind of issue (or you can use `stringBefore` instead of `nodeBefore`): https://github.com/millermedeiros/esformatter/issues/216#issuecomment-76498237

**Comment by e-cloud** [comment]
@bulyshko is this problem still existing?

**Comment by bulyshko** [comment]
@e-cloud Yes, workaround is still used.
